app/models/project.rb

```diff
@@ -131,4 +130,32 @@ def project_events(order_by = "Completed IS NULL OR Completed = '0000-00-00', Co
   def health_status_available?
     return false
   end
+
+  def self.new_order(person)
+    "case when projects.primary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'Loan' then 8
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'BasicProject' then 7
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'Loan' then 6
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'BasicProject' then 5
+
```

Comment:
Next in the list would be be for primary agent id and anything with one of the OPEN_STATUES not 'prospective' and 'active'. Right now all of the remaining statues show up at the bottom of the list, even if the user is the primary agent.

The next two items needed are:
All remaining loans where the user is the primary agent
All remaining basic projects where the user is the primary agent